pkg/apis/core/validation/shoot_test.go

```diff
@@ -790,6 +790,25 @@ var _ = Describe("Shoot Validation Tests", func() {
 
 				Expect(errorList).To(HaveLen(0))
 			})
+
+			It("should not allow changing the enablement of containerd in existing worker pool", func() {
```

Comment:
if my assumption is right that containerd can be enabled, but not disabled then
```suggestion
			It("should not allow containerd to be disabled for existing worker pool", func() {
```